config/general.php

```diff
@@ -30,8 +30,16 @@
         // Path to error templates
         'errorTemplatePrefix' => '_errors/',
 
-        // Allow XML file upload in assets
-        'extraAllowedFileExtensions' => 'xml',
+        // Allow extra file types to be uploaded in assets
+        'extraAllowedFileExtensions' => ['xml', 'css'],
+
+        // List of additional file types Craft should support
+        'extraFileKinds' => [
```

Comment:
What is `extraFileKinds` vs `extraAllowedFileExtensions`?

Reply:
_extraFileKinds_ defines a new file type for when you setup the field. 
_extraAllowedFileExtensions_ makes it available as a choice.